### ISSUE TITLE
[Questions] Why it doesnt overwrite/replace actual uyouEnhanced Youtube app?

### ISSUE THREAD

**Issue by zahid-58** [arichornlover/uYouEnhanced#330]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

I cant update/ overwrite actual uYouEnhanced Youtube app if for example there is a newer version and im signing it then want to install but nothing happens. i‘ve to delete the insalled one and install the update but then my settings or playback history are gone..
Other apps are working. 
Im using an Iphone 15 pro on IOS 17.3 and singning with Signulous.

**Comment by maxexcloo** [comment]
Had this problem also.

**Comment by Hydro1313** [comment]
Same thing happens to me. Unfortunately I deleted the app and installed the latest release and lost all my settings. Not only that, the release is bugged and all of the uYouEnhanced features are not accessible.

**Comment by zahid-58** [comment]
Is there a way to save all the app Data before deleting it? I dont have a jailbreak. Or we sould wait until the bug is fixed..

**Comment by yodaluca23** [comment]
The only reason why it wouldn't overwrite it, is if your using a different signing certificate, or you set a different bundleID, check your signing methods options to make sure it hasn't changed the bundleID...

**Comment by Hydro1313** [comment]
> The only reason why it wouldn't overwrite it, is if your using a different signing certificate, or you set a different bundleID, check your signing methods options to make sure it hasn't changed the bundleID...

Nope, all my apps are signed the same way. I do not change the bundle ID nor do I install as a duplicate. All my apps are installed by default settings.when updating the sideload apps,  the updated version installs over the previous version. 
when I try to install the updated uYouEnhaced, it does nothing. No loading and no errors. The app remains untouched. I have to delete the old version to install the new version. This has been the way for the last 3-4 versions since I started using it.

**Comment by zahid-58** [comment]
> > The only reason why it wouldn't overwrite it, is if your using a different signing certificate, or you set a different bundleID, check your signing methods options to make sure it hasn't changed the bundleID...
> 
> Nope, all my apps are signed the same way. I do not change the bundle ID nor do I install as a duplicate. All my apps are installed by default settings.when updating the sideload apps, the updated version installs over the previous version. when I try to install the updated uYouEnhaced, it does nothing. No loading and no errors. The app remains untouched. I have to delete the old version to install the new version. This has been the way for the last 3-4 versions since I started using it.

Yea can only agree exactly same for me

**Comment by hi-im-karma** [comment]
Is this still an issue? I just started using uyouenhanced and kinda worried what will happen when there's a new version and I update.

**Comment by Hydro1313** [comment]
Yes it is. For what ever reason, this ipa will not install util the old version is deleted. To test that theory, I got a different uYou ipa and it overwrites no problem. When the new uYouEnhanced version comes out, I’ll test it again.

**Comment by hi-im-karma** [comment]
I really hope it won't happen for me. I don't want to loose every setting that I set up to my needs. Setting up everything from scratch every time would be a pain in the ass. 

Is the developer aware of it?

**Comment by zahid-58** [comment]
@arichornlover can u have a look at this ?

**Comment by Hydro1313** [comment]
Just installed this latest version 19.06.02 - 3.0.3 and this time the ipa overwrote the previous version of uYou. All my settings were saved. Hopefully then the next version of uYouEnhanced comes, it will overwrite this current version.

**Comment by zahid-58** [comment]
> Just installed this latest version 19.06.02 - 3.0.3 and this time the ipa overwrote the previous version of uYou. All my settings were saved. Hopefully then the next version of uYouEnhanced comes, it will overwrite this current version.


Im on uYouEnhanced_19.03.2_3.0.1.ipa and the current version didn't overwrite it. 
Anyone else tested it?

**Comment by Hydro1313** [comment]
> > Just installed this latest version 19.06.02 - 3.0.3 and this time the ipa overwrote the previous version of uYou. All my settings were saved. Hopefully then the next version of uYouEnhanced comes, it will overwrite this current version.
> 
> Im on uYouEnhanced_19.03.2_3.0.1.ipa and the current version didn't overwrite it. Anyone else tested it?

My last version of the youtube ipa was 19.06.02_3.0.2 but was not the enhanced version. It was uYou only verson. This version of uYouEnhnaced 19.06.2_3.0.3 had no problem overwriting it. Until the next release comes out, i won't know if the overwite issie is still a problem.
